src/lib.rs

```diff
@@ -308,3 +309,8 @@ pub use crate::ser::to_writer;
 #[cfg(feature = "std")]
 #[doc(inline)]
 pub use crate::value::Value;
+
```

Comment:
Why do you need these use statements? lib.rs should only include the public re-exports.